WeScan/Common/CaptureSession.swift

```diff
@@ -17,14 +17,14 @@ final class CaptureSession {
     var isEditing: Bool
     
     /// Whether auto scan is enabled or not
-    var autoScanEnabled: Bool
+    var isAutoModeEnabled: Bool
```

Comment:
We don't mention tap to focus in the documentation for this variable, I think if we are going to go with this flag we should explain when they want this to be false so it's clear how to opt into the tap to focus behavior.

Reply:
Sure, will do.

Reply:
Resolved: 9d1e1e4